workflows_core/dataset/field.py

```diff
@@ -155,6 +155,31 @@ def get_centroids(self, alias: str):
             "`insert_centroids` not available for non vector_fields"
         )
 
+    def get_keyphrase(self, alias: str, keyphrase_id: str):
```

Comment:
do we need these?

Reply:
we dont' need this for now. but not sure if we would need them later so I put it here.